coderd/telemetry/telemetry.go

```diff
@@ -353,9 +353,6 @@ func (r *remoteReporter) createSnapshot() (*Snapshot, error) {
 		users := database.ConvertUserRows(userRows)
 		var firstUser database.User
 		for _, dbUser := range users {
-			if dbUser.Status != database.UserStatusActive {
-				continue
-			}
```

Comment:
It looks like this is from your previous pr https://github.com/coder/coder/pull/13613